README.md

```diff
@@ -37,7 +37,7 @@ The fields in the table below can be used in these parts of STAC documents:
 
 | Field Name       | Type                     | Description |
 | ---------------- | ------------------------ | ----------- |
-| proj:epsg        | string   | [EPSG code](http://www.epsg-registry.org/) of the datasource; Maintained for backwards compatability. Please use `proj:authority` and `proj:code`. |
+| proj:epsg        | integer   | [EPSG code](http://www.epsg-registry.org/) of the datasource; Maintained for backwards compatability and will be deprecated in V2.0.0. Please use `proj:authority` and `proj:code`. |
```

Comment:
integer|null

By the way, is there a way to explicitly say "no projection" with code and authority?